spec/aged_brie_spec.js

```diff
@@ -1,52 +1,57 @@
 describe("Aged Brie", function() {
+    const anonymousName = "Aged Brie";
```

Comment:
I don't use real values in my anonymous values, as it can lead to confusion: is this for Aged Brie, or do we not care what value is used? Additionally, if there's an AgedBrie class, why pass in the name at all?

Finally, I wouldn't place these here, as the point of anonymous values is that they are throwaway values. For strings, you can use a hard-coded value of `"anonymousName"`, for example; I use named consts within tests for things like ints that don't convey any meaning in and of themselves.